content/intro-to-storybook/svelte/en/get-started.md

```diff
@@ -0,0 +1,192 @@
+
+```javascript
+function sum(a, b) {
+  return a + b;
+}
+describe("Sample Test", () => {
+  it("should return 3 as the result of the function", () => {
+    // set timeout to prevent false positives with tests
+    expect(sum(1, 2)).toBe(3);
+  });
+});
+
+```
+
+Now we can quickly check that the various environments of our application are working properly:
+
+```bash
+# Run the test runner (Jest) in a terminal:
+npm run test
+
+# Start the component explorer on port 9009:
+npm run storybook
+
+# Run the frontend app proper on port 5000:
+npm run dev
+```
+
+Our three frontend app modalities: automated test (Jest), component development (Storybook), and the app itself.
+
+![3 modalities](/intro-to-storybook/app-three-modalities-svelte.png)
+
+Depending on what part of the app you’re working on, you may want to run one or more of these simultaneously. Since our current focus is creating a single UI component, we’ll stick with running Storybook.
+
+## Reuse CSS
+
+Taskbox reuses design elements from the GraphQL and React Tutorial [example app](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), so we won’t need to write CSS in this tutorial. Copy and paste [this compiled CSS](https://github.com/chromaui/learnstorybook-code/blob/master/src/index.css) into the src/index.css file.
+
+We'll need a new dependency to bundle the copied css file into a separate file, leaving any component styles separated and with that avoiding any styling issues. Run the following command:
+
+```bash
+npm install -D rollup-plugin-css-only
+```
+
+Modify your `rollup.config.js` to the following:
+
+```js
+// ./rollup.config.js
+import svelte from 'rollup-plugin-svelte';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import livereload from 'rollup-plugin-livereload';
+import { terser } from 'rollup-plugin-terser';
+import rollup_start_dev from './rollup_start_dev';
+import css from 'rollup-plugin-css-only';
+
+const production = !process.env.ROLLUP_WATCH;
+
+export default {
+	....
+	plugins: [
+		....
+		css({output: 'public/index.css'}), //bundles the copied css into the app 
+		....
+	],
+};
+
+```
+
+Modify `public/index.html` to reference the css file we just added, by adding the following:
+
+```html
+<link rel="stylesheet" href="index.css">
+```
+
+And remove the following tag:
+
+```html
+<link rel='stylesheet' href='/global.css'>
+```
+<div class="aside">You can also safely remove the global.css file aswell as it will not be used with this tutorial</div>
+
+![Taskbox UI](/intro-to-storybook/ss-browserchrome-taskbox-learnstorybook.png)
+
+<div class="aside">
+If you want to modify the styling, the source LESS files are provided in the GitHub repo.
+</div>
+
+## Add assets
+
+We also need to add the font and icon [directories](https://github.com/chromaui/learnstorybook-code/tree/master/public) to the `public/` folder.
```

Comment:
This one was one of those that got me wondering. But i went with the current flow of documentation. My initial thought that could be applied here and then moved into the other tutorials is to apply the same "strategy" used for the design systems tutorial, more specifically in [the architecting systems section](https://www.learnstorybook.com/design-systems-for-developers/react/en/architecture/) under "Create an inventory" . What do you think?

Reply:
I've updated it to match the design systems wording and structure. One thing i would like for you to check is the urls that i've used in conjunction with svn. I don't have svn installed at this moment so i was unable to test them. I changed them because the ones you supplied were throwing a 401 error with github.

Reply:
I think the URLs only work over svn:

<img width="799" alt="image" src="https://user-images.githubusercontent.com/132554/69112077-8db44400-0ad3-11ea-9f35-a8499d35331a.png">

Reply:
Updated the links with the latest commit.